packages/salesforcedx-apex-debugger/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@salesforce/salesforcedx-apex-debugger",
+  "displayName": "Apex Debugger Adapter",
+  "description": "Implements VSCode debug protocol for Apex Debugger",
+  "version": "40.5.1",
+  "publisher": "salesforce",
+  "license": "BSD-3-Clause",
+  "engines": {
+    "vscode": "^1.15.0"
```

Comment:
OK. Per our discussion, we probably want to bump up the other extensions to require VS Code 1.15. However, we should probably wait until 1.16 comes out to bump the rest to 1.15. Once we bump this requirement, anyone on the older version won't be able to install this.

See vscode/issues/32590 (I am not going to link to it directly since that would then publicly put this particular PR on that issue, per the way GitHub works, causing additional noise).